documentation/IDTA-01002-3/modules/ROOT/pages/Annex/IDTA-01002_SerializationModifierExamples.adoc

```diff
@@ -118,52 +119,15 @@ If applied to the Submodel:
         "value": "0173-1#01-AFZ615#016"
     } ],
     "type": "ExternalReference"
-  },
-  "submodelElements": [ {
-    "modelType":  "SubmodelElementCollection",
-    "idShort": "RotationSpeed",
-    "semanticId": {
-      "keys": [ {
-          "type": "GlobalReference",
-          "value": "http://purl.org/iot/vocab/iot-taxonomy-lite#RotationalSpeed"
-      } ],
-      "type": "ExternalReference"
-    },
-    "value": [ {
-        "modelType": "Property",
-        "idShort": "MaxRotationSpeed",
-        "category": "PARAMETER",
-        "semanticId": {
-          "keys": [ {
-              "type": "ConceptDescription",
-              "value": "0173-1#02-BAA120#008"
-          } ],
-          "type": "ExternalReference"
-        },
-        "valueType": "xs:int"
-      } ]
-  } ]
+  }
 }
-
 ----
 
-a|
-If applied to the Submodel:
+If applied to the SubmodelElementCollection, i.e., idShortPath “RotationSpeed”:
 
 [source,json,linenums]
 ----
 {
-  "modelType": "Submodel",
-  "id": "http://i40.customer.com/type/1/1/7A7104BDAB57E184",
-  "idShort": "TechnicalData",
-  "semanticId": {
-    "keys": [ {
-        "type": "GlobalReference",
-        "value": "0173-1#01-AFZ615#016"
```

Comment:
For Technical Data it would be https://admin-shell.io/ZVEI/TechnicalData/Submodel/1/2
or change to Handover Docmentation, there it is an IRDI, see line 119